src/main/java/com/github/dockerjava/api/model/ExposedPorts.java

```diff
@@ -56,8 +56,8 @@ public void serialize(ExposedPorts exposedPorts, JsonGenerator jsonGen, Serializ
 
     public static class Deserializer extends JsonDeserializer<ExposedPorts> {
         @Override
-        public ExposedPorts deserialize(JsonParser jsonParser, DeserializationContext deserializationContext)
-                throws IOException, JsonProcessingException {
+        public ExposedPorts deserialize(JsonParser jsonParser, DeserializationContext deserializationContext) throws IOException,
+                JsonProcessingException {
```

Comment:
:-1: i usually using first variant because it more human readable. On one line arguments and on other exceptions. You don't need keep in memory the whole list of args/exceptions while reading it. 
Never checkstyled such things.